nixos/tests/zipline.nix

```diff
@@ -5,6 +5,9 @@
   meta.maintainers = with lib.maintainers; [ defelo ];
 
   nodes.machine = {
+    # Testing with a CPU without SSE 4.2 support
+    # to ensure native libvips is used
+    virtualisation.qemu.options = [ "-cpu core2duo" ];
```

Comment:
This doesn't seem to work on `aarch64-linux` unfortunately (see [ofborg logs](https://logs.ofborg.org/?key=nixos/nixpkgs.415805&attempt_id=370b8c5a-1a3e-4827-ac8d-c199e8858203)). Since this is an x86 issue anyway (iiuc) I would suggest to use something like `lib.mkIf pkgs.stdenv.hostPlatform.isx86 [ "-cpu core2duo" ]` here.